tap-setup-scripts/src/tap-main.sh

```
@@ -37,6 +37,7 @@ function tapInstallMain {
   case $CLUSTER_NAME_SUFFIX in
     build)
       kubectl create -f $RESOURCES/tap-gui-viewer-service-account-rbac.yaml || true
+      kubectl create -f $RESOURCES/tap-gui-viewer-secret.yaml || true
```

*question*: Not sure why we do a `kubectl create ... || true` here? My assumption is, we do it to not fail if the secret already exists. But wouldn't a `kubdectl apply ...` be better and not swallow potential issues/errors?

---

These would go away, I guess, if we'd add the token secret as an overlay to `tap-gui`, see other comment.

---

Yes, we can use "apply" instead of "create". I  have not seen any significant differences in the behaviors for all our yaml resources till now.

---

I am always uneasy when I see something like `|| true` without explaination / good reason / ... because they just swallow all errors, also the ones I potentially don't want to ginore.